scrapi/harvesters/ut_chattanooga.py

```diff
@@ -0,0 +1,29 @@
+'''
+Harvester for the UTC Scholar for the SHARE project
+
+Example API call: http://scholar.utc.edu/do/oai/?verb=ListRecords&metadataPrefix=oai_dc
+'''
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class Ut_chattanoogaHarvester(OAIHarvester):
+    short_name = 'ut_chattanooga'
+    long_name = 'University of Tennessee at Chattanooga'
+    url = 'http://scholar.utc.edu'
+
+    base_url = 'http://scholar.utc.edu/do/oai/'
+    property_list = ['type', 'format', 'identifier', 'date', 'source', 'setSpec', 'publisher', 'language', 'rights']
+
+    approved_sets = [
+        u'honors-theses',
+        u'theses',
+        u'student-research',
+        u'ebm-symposium',
+        u'conferences',
+        u'journals',
+        u'jafh'
+    ]
```

Comment:
@erinspace they had sent us the following:
Approved sets:
[u'honors-theses', u'theses']

based on your recent comment, should I remove the other approved sets I through in there? I had added them since I thought that was how to handle the error that the harvester pops out.